scripts/kind-e2e/lib_operator_verify_subm.sh

```diff
@@ -371,7 +371,19 @@ function verify_subm_routeagent_container() {
 }
 
 function verify_subm_broker_secrets() {
-  # Show all SubM secrets
+  # Wait for secrets to be created
+  COUNT="0"
```

Comment:
This would be better as `SECONDS` because that’s what it measures; `COUNT` gives the impression it’s an iteration count. (Same goes for the subsequent uses.)

Reply:
Done, thanks:)